presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -340,6 +328,18 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         this.optimizers = builder.build();
     }
 
+    @PostConstruct
+    public void initialize()
+    {
+        stats.export(exporter);
+    }
+
+    @PreDestroy
+    public void destroy()
+    {
+        stats.unexport(exporter);
+    }
```

Comment:
I am not sure if we should touch this at all. They are special, so maybe they should be before the constructor.